src/components/MainNavigation/MainNavigation.tsx

```diff
@@ -60,7 +60,9 @@ export const MainNavigation: FC<FlexProps> = (props) => {
 									{t('label.github')}
 								</MenuItem>
 								<MenuDivider />
-								<MenuItem onClick={handleLogout}>Log out</MenuItem>
+								<MenuItem as={NextLink} href="/" onClick={handleLogout}>
```

Comment:
I assume you want to redirect a user to home page after he logged out? You can do that in `handleLogout` 

```tsx
const router = useRouter();

const handleLogout = () => {
  logout();
  router.push("/");
}
```